ember-app/package.json

```diff
@@ -11,7 +11,7 @@
     "start": "ember server",
     "build": "ember build",
     "test": "ember test",
-    "postinstall": "pwd && ../node_modules/.bin/bower install"
+    "postinstall": "node ../node_modules/bower/bin/bower install"
```

Comment:
I wasn't aware that npm on windows and npm on unix differed in this convention. 

This might not work for a heroku deployment